lib/modules/versioning/devbox/readme.md

```diff
@@ -0,0 +1,2 @@
+Devbox's Nixhub uses fairly strict versioning, no ranges are permitted.
```

Comment:
I think this is incorrect. `1` and `1.2` are examples of ranges

Reply:
I think I meant things like ~, ^ and >= not being allowed. I'll update it to that

Reply:
Ok that's updated now, let me know if that makes sense now